### PR TITLE
Feature/homepage styling

### LINKED ISSUE #4: Homepage styling
Style the Homepage

### LINKED ISSUE #16: Nav bar styling
Complete styling for nav bar

### PR DESCRIPTION
**Pull request by michannstoner**
##### Description
- Installed sass 
- Added styling to homepage

##### What's changed & where should the reviewer start?
- Home.js / Home.scss
- Nav.js / Nav.scss
- styles directory and all corresponding files 

##### Relevant Screenshots/GIFs
Screen Shot 2021-07-10 at 3 17 17 PM

#### Type of change
What is it?
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring

### Checklist:
- [x] My code follows the guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream module

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -8,9 +8,13 @@
...
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <title>React App</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Bitter&family=Work+Sans&display=swap" rel="stylesheet">
+    <title>Roomies</title>
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (AndieDrew) [inline public/index.html:20]: Nice adding some custom fonts!

File: src/components/Home/Home.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  border-radius: 15px;
+  @include flex(row);
+}
+
+.welcome-section {
+  @include flex(column);
+  justify-content: center;
+  align-items: center;
+  margin-top: 1em;
+}
```
Comment (AndieDrew) [inline src/components/Home/Home.scss:25]: Great job setting things up as scss! Thought we'd have to refactor our css later but might as well get it right from the start!

**Review by Thee-Dust — COMMENTED** [review]
LGTM!

File: src/styles/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+$darkGreen: #157A6E;
+$mediumGreen: #499F68;
+$lightGreen: #77B28C;
+$burntOrange: #B4654A;
+$primaryWhite: #FFFFFF;
+
+$primaryFont: 'Work Sans', sans-serif;
+$accentFont: 'Bitter', serif;
```
Comment (AndieDrew) [inline src/styles/_variables.scss:8]: Great job creating reusable variables!

**Review by AndieDrew — APPROVED** [review]

**Pull request merged**